crates/core/tedge_actors/src/servers/mod.rs

```diff
@@ -50,52 +50,59 @@
 //! To be used as an actor, a `Server` is wrapped into a [ServerActor](crate::ServerActor)
 //!
 //! ```
-//! # use tedge_actors::{Actor, Builder, NoConfig, MessageReceiver, Sender, ServerActor, SimpleMessageBox, SimpleMessageBoxBuilder};
+//! # use tedge_actors::{Actor, Builder, NoConfig, MessageReceiver, Sender, ServerActorBuilder, ServerConfig, Sequential};
 //! # use crate::tedge_actors::examples::calculator_server::*;
 //! #
 //! #[cfg(feature = "test-helpers")]
 //! # #[tokio::main]
 //! # async fn main_test() {
 //! # use tedge_actors::test_helpers::ServiceProviderExt;
 //! #
-//! // As for any actor, one needs a bidirectional channel to the message box of the server.
-//! let mut actor_box_builder = SimpleMessageBoxBuilder::new("Actor", 10);
-//! let mut client_box = actor_box_builder.new_client_box(NoConfig);
-//! let server_box = actor_box_builder.build();
-//!
-//! // Create an actor to handle the requests to a server
-//! let server = Calculator::default();
-//! let mut actor = ServerActor::new(server, server_box);
-//!
-//! // The actor is then spawn in the background with its message box.
+//! // As for any actor, one needs a handle to the message box of the server.
+//! // The simpler is to use a builder.
+//! let config = ServerConfig {
+//!             capacity: 16,
+//!             max_concurrency: 4,
+//! };
+//! let actor = ServerActorBuilder::new(Calculator::default(), &config, Sequential);
```

Comment:
question: little surprising to see `max_concurrency: 4` along with `Sequential`. Because `Sequential` creates a normal logging receiver instead of a `ConcurrentServerMessageBox`, I understand that `max_concurrency` is ignored in this case?

Reply:
Yes, this `ServerConfig` definitely deserves some clean-up.